core/domain/user_jobs_one_off_test.py

```diff
@@ -589,6 +598,58 @@ def test_weekly_stats_for_multiple_explorations(self):
             }
         }])
 
+    def test_stats_for_multiple_weeks(self):
```

Comment:
ah yes. it should be 2. I don't have time to look at this right now, but will try to take a look tomorrow.